include/imex/Conversion/GPUToSPIRV/GPUToSPIRVPass.h

```diff
@@ -0,0 +1,36 @@
+//===- GPUToSPIRVPass.h - GPUToSPIRV conversion  ---------------*- C++ -*-===//
+//
+// Copyright 2022 Intel Corporation
+// Part of the IMEX Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file defines the GPUToSPIRV conversion,
+///
+//===----------------------------------------------------------------------===//
+
+#ifndef IMEX_GPUTOSPIRV_PASS_H_
```

Comment:
Side question: are we going with include guards in refactor or with `#pragma once`?

Reply:
so far I have seen upstream is following include guards rather than #pragma once, and refactor is also following the same I believe, @fschlimb/others can clarify if there are other plans?

Reply:
Yes, I also see upstream using #ifndef guards. They are more flexible, more reliable and standard C++. I guess both approaches would be acceptable in most cases, though. I don't know if there was a explicit decision made in upstream but I suggest we follow what happens there (only clang seems to sometimes use the pragma).